### DISCUSSION TITLE
After Adding PATH, .zprofile:1 not found?

### DISCUSSION THREAD

**Discussion by fourteenthdee** [Homebrew/discussions#951] [Getting started]
Hello, all.

Every time I open the Terminal, I get an error:

`/Users/username/.zprofile:1: no such file or directory: /Users/homefolder/.zprofile`

The content of my .zprofile is this:

`echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
eval $(/opt/homebrew/bin/brew shellenv)`

What I doing wrong here?

Thank you!

**Comment by gromgit** [comment]
It looks like what you did was to add:
```
echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
```
to your .zprofile, so every time you open a new terminal session, yet another `shellenv` line would be added to your .zprofile, _ad infinitum_, and you're now pointlessly loading your Homebrew environment multiple times.

The "Next Steps" instruction you got at the end of the Homebrew installer was to _run_ the above command in your Terminal, which would insert only:
```
eval $(/opt/homebrew/bin/brew shellenv)
```
into your .zprofile, and every time you open a new terminal session, `brew shellenv` would be run only once.

You'll now need to manually clean up your .zprofile, to remove all `shellenv` lines except one instance that looks _exactly_ like the second command I just posted.

**Reply by gromgit** [reply]
As for your original error:
> ```
> /Users/username/.zprofile:1: no such file or directory: /Users/homefolder/.zprofile
> ```

`/Users/username/.zprofile` and `/Users/homefolder/.zprofile` are different files, and I'm guessing that:
* you obfuscated `/Users/username`
* you did _not_ obfuscate `/Users/homefolder`
* the `/Users/homefolder` directory really doesn't exist on your machine

**Reply by fourteenthdee** [reply]
That was exactly it.   missed the "next steps" instructions on the first go round and misunderstood the remedy.  Thanks sorting it out for me!
